src/librbd/api/Mirror.cc

```diff
@@ -2976,23 +3015,53 @@ int create_orphan_group_snapshot(IoCtx& group_ioctx,
                  << ", group_snap_id=" << group_snap_id
                  << dendl;
 
+  librados::Rados rados(group_ioctx);
+  int8_t require_osd_release;
+  bool osds_upgraded = false;
```

Comment:
This boolean's name is not intuitive.  A boolean named `osds_upgraded` to me indicates that OSDs have been either upgraded or not. This variable looks like a boolean to determine whether the minimum version of OSDs in the cluster is tentacle. So maybe name it as , `is_min_compatible_osd_release_atleast_tentacle` ?

However, I see this boolean being used in only other place. Can we get rid of this boolean, and just use  `require_osd_release >= CEPH_RELEASE_TENTACLE` , where this boolean is currently used?